internals-js/src/buildSchema.ts

```diff
@@ -299,7 +299,9 @@ function buildNamedTypeInner(
       const enumType = type as EnumType;
       for (const enumVal of definitionNode.values ?? []) {
         const v = enumType.addValue(enumVal.name.value);
-        v.description = enumVal.description?.value;
+        if (enumVal.description) {
```

Comment:
Is this a change? Behavior looks the same to me.

Reply:
See comment on types below. I actually don't think the change is necessary in that case of enum values, but just made it to mirror the other similar cases in the file.